src/views/WalletSelect.svelte

```diff
@@ -29,22 +29,42 @@
     WalletSelectModule,
     WalletInterface
   } from '../interfaces'
+  import { STORAGE_KEYS } from '../constants'
 
   export let module: WalletSelectModule = {
     heading: '',
     description: '',
+    termsOfServiceUrl: '',
+    privacyPolicyUrl: '',
     wallets: []
   }
 
   let modalData: WalletSelectModalData | null
   let showWalletDefinition: boolean
   let walletAlreadyInstalled: string | undefined
-  let installMessage: string | undefined
+  let installMessage: string
 
   let selectedWalletModule: WalletModule
 
   const { mobileDevice, os } = get(app)
-  let { heading, description, explanation, wallets } = module
+  let {
+    heading,
+    description,
+    explanation,
+    termsOfServiceUrl,
+    privacyPolicyUrl,
+    wallets
+  } = module
+
+  let showTermsOfService: boolean = !!(termsOfServiceUrl || privacyPolicyUrl) && !get(app).termsAgreed
+
+  app.subscribe(({ termsAgreed }) => {
```

Comment:
```suggestion
const unsubscribe = app.subscribe(({ termsAgreed }) => {
```

I think we will need the unsubscribe method to unsub when this component is destroyed to avoid a memory leak